Toolkit/src/osh/ui/view/video/osh-UI-FFMPEGView.js

```diff
@@ -244,64 +245,30 @@ OSH.UI.FFMPEGView = OSH.UI.View.extend({
      * @param callback
      */
     initFFMPEG_DECODER_WORKER: function (callback) {
-        this.worker = new Worker(window.OSH.BASE_WORKER_URL+'/osh-UI-FFMPEGViewWorker.js');
-
+        this.worker = new Worker('js/workers/osh-UI-FFMPEGViewWorker.js');
```

Comment:
I think we need to keep the window.OSH.BASE_WORKER_URL+ here so that the WebWorker is loaded from the configured folder

Reply:
I've restored the old code